src/content/reference/device-os/firmware.md

```diff
@@ -174,11 +271,15 @@ Up to 15 cloud functions may be registered and each function name is limited to
 **Note:** Only use letters, numbers, underscores and dashes in function names. Spaces and special characters may be escaped by different tools and libraries causing unexpected results.
 A function callback procedure needs to return as quickly as possible otherwise the cloud call will timeout.
 
+The callback function is called application loop thread context, between calls to loop(), and during Particle.process().
```

Comment:
`in` missing, while in `delay()` as well.

Reply:
Fixed